services/workflow-manager/app/api/workflow.py

```diff
@@ -21,19 +36,115 @@
 # 404 = Not found
 
 
+SEQUENCE_MANAGER_URI = "host.docker.internal:8003"
+EXAM_MANAGER_URI = "host.docker.internal:8004"
+
+
 class RecoJob(BaseModel):
     """RecoJob is a pydantic model for a reco job."""  # noqa: E501
 
     record_id: int
     input: StrictStr
 
 
+class TaskEvent(BaseModel):
```

Comment:
This should be defined in the base image

Reply:
Right, moved to base image